src/raw/mod.rs

```diff
@@ -46,14 +46,15 @@ unsafe fn offset_from<T>(to: *const T, from: *const T) -> usize {
 // outweighs any gains over the generic implementation.
 #[cfg(all(
     target_feature = "sse2",
-    any(target_arch = "x86", target_arch = "x86_64")
+    any(target_arch = "x86", target_arch = "x86_64"),
+    not(miri)
 ))]
 #[path = "sse2.rs"]
 mod imp;
-#[cfg(not(all(
+#[cfg(any(not(all(
     target_feature = "sse2",
     any(target_arch = "x86", target_arch = "x86_64")
-)))]
+)), miri)]
```

Comment:
There are 4 opening brackets in line 54 but only 3 closing in line 57.

Reply:
Weird, you are right, yet it compiles fine with only 3 closing brackets =/ 
Maybe we should fill an issue upstream.